mitodl/mitxonline
# Added data model for ProgramRequirement

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #1066 

#### What's this PR do?
Adds a data model for program requirements

#### How should this be manually tested?
- Migrations should run and tests should pass
- You should be able to edit the requirements for a program:
  - Add, remove, reorder top-level sections (e.g. "Required Courses" in the screenshot)
  - Add, remove, reorder second-level requirements (courses or operators)
  - Add, remove, reorder third-level courses under second-level operators

#### Screenshots

![Screenshot 2022-10-26 at 11-26-00 no-id Data Economics and Development Policy Change program Django site admin](https://user-images.githubusercontent.com/28598/198068673-c712064c-271a-442f-9869-6932097def72.png)
